linera-chain/src/data_types.rs

```diff
@@ -395,16 +399,16 @@ impl OutgoingMessage {
     }
 }
 
-/// A [`Block`], together with the outcome from its execution.
+/// A [`Proposal`], together with the outcome from its execution.
 #[derive(Debug, PartialEq, Eq, Hash, Clone, Serialize, Deserialize, SimpleObject)]
 pub struct ExecutedBlock {
```

Comment:
Maybe `ProposalWithOutcome`. Or even inline it, if it's not used in many places anymore?